src/Psy/Configuration.php

```diff
@@ -721,6 +721,14 @@ public function setTabCompletion($tabCompletion)
     }
 
     /**
+     * @param array $matchers
+     */
+    public function setTabCompletionMatchers(array $matchers)
+    {
+        $this->tabCompletionMatchers = array_merge($this->tabCompletionMatchers, $matchers);
+    }
```

Comment:
Sorry my bad. I tried to add matchers via config and threw an error. I will double check the calls and maybe a test for that.